test-utils/testlib/src/runtime_utils.rs

```diff
@@ -27,18 +24,11 @@ pub fn evm_account() -> AccountId {
 }
 
 pub fn default_code_hash() -> CryptoHash {
-    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-    path.push("../../runtime/near-vm-runner/tests/res/test_contract_rs.wasm");
-    let genesis_wasm = fs::read(path).unwrap();
-    hash(&genesis_wasm)
+    hash(&near_test_contracts::rs_contract())
 }
 
 lazy_static::lazy_static! {
-    static ref DEFAULT_TEST_CONTRACT_HASH: CryptoHash = hash(&DEFAULT_TEST_CONTRACT);
-}
-
-lazy_static_include::lazy_static_include_bytes! {
-    DEFAULT_TEST_CONTRACT => "../../runtime/near-vm-runner/tests/res/test_contract_rs.wasm"
+    static ref DEFAULT_TEST_CONTRACT_HASH: CryptoHash = hash(near_test_contracts::rs_contract());
 }
```

Comment:
This seems to be some pre-existing duplication: `default_code_hash` and `DEFAULT_TEST_CONTRACT` compute the same thing. Left as is, as out of scope for the PR.